ibis/backends/tests/test_aggregation.py

```diff
@@ -312,7 +323,7 @@ def test_approx_median(backend, alltypes, df):
         )
     ],
 )
-@mark.notimpl(["datafusion"])
+@mark.notimpl(["datafusion", "duckdb"])
```

Comment:
This function should be expressible as `string_agg` (https://duckdb.org/docs/sql/aggregates).